### ISSUE TITLE
404-page not found

### ISSUE THREAD

**Issue by codewithmide** [weavedb/weavedb-docs#13]
The current WeaveDB docs page does not have a 404 page. I would like to make this changes by making a contribution.

**Comment by drumfeet** [comment]
It uses the default. Just a simple message that page could not be found

**Comment by codewithmide** [comment]
> It uses the default. Just a simple message that page could not be found

Yeah, I noticed that also. I'm only suggesting that we have a more personalized 404-page for weaveDB that includes a call to action button that takes users back to the homepage when clicked.

If that is fine by you, Let me know.

**Comment by drumfeet** [comment]
Yep, that is fine
